go/vt/vtgate/executor.go

```diff
@@ -901,7 +901,7 @@ func (e *Executor) showVitessReplicationStatus(ctx context.Context, filter *sqlp
 	for _, s := range status {
 		for _, ts := range s.TabletsStats {
 			// We only want to show REPLICA and RDONLY tablets
-			if ts.Tablet.Type != topodatapb.TabletType_REPLICA && ts.Tablet.Type != topodatapb.TabletType_RDONLY {
+			if ts.Target.TabletType != topodatapb.TabletType_REPLICA && ts.Target.TabletType != topodatapb.TabletType_RDONLY {
```

Comment:
I did confirm that `Target.TabletType` is what's being used elsewhere when examining/showing the type for tablets in the healthcheck cache. So this part seems right.

Following on from that, I think that we should change all usage of `ts.Tablet.*` in this function to `ts.Target.*`. For example, below we should change `ts.Tablet.Keyspace` to `ts.Target.Keyspace` and `ts.Tablet.Shard` to `ts.Target.Shard`.